lib/UserAgentApplication.ts

```diff
@@ -193,12 +180,18 @@ namespace Msal {
                 this._tokenReceivedCallback = tokenReceivedCallback;
             }
 
-            this.redirectUri = window.location.href.split("?")[0].split("#")[0];
```

Comment:
Why not use default parameters instead of adding another property and assignment?